docs/manual/manual_stock_led.rst

```diff
@@ -0,0 +1,251 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _manual_led_support:
+
+***********************************************
+Stock LED Support (White and RGB LED Control)
+***********************************************
+All Vertiq Gen 2 modules support use of our LED add-on board available with Vertiq's Pro Kit. For information about purchasing a Pro Kit, or for information about 
+purchasing more LED add-on boards, please contact us at support@vertiq.co. Please refer to your module's family page to see if it belongs to Gen 1 or Gen 2.
```

Comment:
May be better to use info@vertiq.co here, and keep support just for support stuff